cli/lib/kontena/cli/helpers/health_helper.rb

```diff
@@ -49,5 +49,49 @@ def node_health(node, grid_health)
         return node['connected'] ? :ok : :offline
       end
     end
+
+    # @return [Symbol]
+    # @return [String]
+    def node_etcd_health(node_health)
```

Comment:
It's used for both `kontena node health` and `kontena etcd health`, so it makes sense here... OTOH the `time_since` should really be elsewhere, it's just stuffed in here because it's used for both `kontena node list` and `kontena node health`.